selenium_tests/course_info/course_info_is_search_page_loaded_test.py

```diff
@@ -0,0 +1,28 @@
+import unittest
+
+from selenium_tests.course_info.course_info_base_test_case import CourseInfoBaseTestCase
+from selenium_tests.course_info.page_objects.course_info_parent_page_object import CourseInfoParentPage
+from selenium_tests.course_info.page_objects.course_info_search_page_object import CourseSearchPageObject
+
+
+class CourseInfoIsSearchPageLoadedTest(CourseInfoBaseTestCase):
+
+    def test_is_loaded(self):
+        """
+        Verify  that the course info search page is loaded by checking against page title
+        """
+        driver = self.driver
+        parent_page = CourseInfoParentPage(driver)
+        search_page = CourseSearchPageObject(driver)
```

Comment:
The `select_course_info_link()` call below (line 19) is returning the PO for you, so I think this line isn't needed.